modules/UI/UI.js

```diff
@@ -590,6 +590,21 @@ UI.updateUserRole = user => {
 };
 
 /**
+ * Updates the user status.
+ *
+ * @param {JitsiParticipant} user - The user which status we need to update.
+ * @param {string} status - The new status.
+ */
+UI.updateUserStatus = (user, status) => {
+    let displayName = user.getDisplayName();
+    messageHandler.notify(
+        displayName, '', 'connected', "dialOut.statusMessage",
+        {
+            status: UIUtil.escapeHtml(status)
+        });
+};
```

Comment:
Good point. However I will actually do the opposite and make the statusMessage string not depending on the dialOut, so that we can use it for something else if we decide. Right now it's pretty generic by design.